src/app/app.component.html

```diff
@@ -1,20 +1,99 @@
+                <label> Branch
+                    <input type="text" formControlName="branch">
+                </label>
+                <div class="alert" *ngIf="!rForm.controls['branch'].valid && rForm.controls['branch'].touched">Branch name must present</div>
+
+                <label>Year of pass
+                    <input type="text" formControlName="yop">
+                </label>
+                <div class="alert" *ngIf="!rForm.controls['yop'].valid && rForm.controls['yop'].touched">A valid Year of pass must present ex 2016</div>
+
+                <label>Place
+                    <input type="text" formControlName="place">
+                </label>
+                <div class="alert" *ngIf="!rForm.controls['place'].valid && rForm.controls['place'].touched">Place must present</div>
+
+
+                <label>Mobile Number
+                    <input type="text" formControlName="mobilenum">
+                </label>
+                <div class="alert" *ngIf="!rForm.controls['mobilenum'].valid && rForm.controls['mobilenum'].touched">Mobile Number is 10 digit number</div>
+
+                <label>Registration Number
+                    <input type="text" formControlName="regnum">
+                </label>
+                <div class="alert" *ngIf="!rForm.controls['regnum'].valid && rForm.controls['regnum'].touched">Registration Number is 12 digit number</div>
+
+                <label>Date Of Birth
+                    <input type="date" formControlName="dob">
+                </label>
+                <div class="alert" *ngIf="!rForm.controls['dob'].valid && rForm.controls['dob'].touched">date of birth MM/DD/YYYY format</div>
+
+                <label>Email Id
+                    <input type="text" formControlName="emailid">
+                </label>
+                <div class="alert" *ngIf="!rForm.controls['emailid'].valid && rForm.controls['emailid'].touched">Valid Id Exaple raj123@xyz.com </div>
+
+                <label>Description
+                    <input type="text" formControlName="description">
+                </label>
+                <div class="alert" *ngIf="!rForm.controls['description'].valid && rForm.controls['description'].touched">Description must be min 30 max 500</div>
+
+                <input type="submit" class="button expanded " value="Submit Form" [disabled]="!rForm.valid">
+
+            </div>
+        </div>
+    </form>
 </div>
-<h2>Here are some links to help you start: </h2>
-<ul>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://angular.io/tutorial">Tour of Heroes</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://github.com/angular/angular-cli/wiki">CLI Documentation</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
-  </li>
-</ul>
 
+<div class="infotable">
```

Comment:
the whole table section needs to reworked upon. This is not what we are looking for